docs/en_US/updating/futurerestore.md

```diff
@@ -33,7 +33,7 @@ You **must** make sure that the latest SEP/BB is compatible to the version you a
   - This should be the same iOS version as your blob
 - On Windows, make sure you have [iTunes](https://www.apple.com/itunes/) installed
   - Scroll down and select the other Windows build as the Windows Store version will not work
-- If on an A10-A11 device on 15.0-15.1.1 **or** an A12+ device on 15.0-15.5b4
+- If on an A10-A11 device on 15.0-15.1.1
```

Comment:
Wouldn’t it make more sense to change this to “If using an A10 device on 15.0-15.1.1” given that all A11 devices got 16.x

Reply:
It would, looks like I just missed that part (even though I reviewed it like 3 times lol)